### DISCUSSION TITLE
The response HTML of the nuxt server is different from that displayed on the page

### DISCUSSION THREAD

**Discussion by sunny-fun** [nuxt/nuxt#10642] [Questions (Nuxt 2)]
In my blog website, I used marked to convert the articles in markdown format stored by me into HTML. There are some codes, but I found that when the code is not a single JavaScript, it will not be highlighted. However, I checked the source code of the web page and found that the nuxt server sent the highlighted code，

telement vue

`<div id="parse-md" class="markdown-body line-numbers" v-html="md(content)"></div>`

script
`
const marked = require('marked')
marked.setOptions({
  highlight: function (code) {
    return highlight(code, languages.js); // languages.<insert language> to return html with markup
  }
});
 async asyncData({ params, $axios, store, redirect }) {
    $axiosgetdata...
  },
 methods: {
 md(text) {
      if (!text) return
      return marked.parse(text);
    },
 }
`

Nuxt server response HTML source code：
markdown: 
````javascript console.log('test')```

When the code in MD is JavaScript, the page can be displayed normally

![Snipaste_2022-08-12_21-23-47](https://user-images.githubusercontent.com/55755493/184363375-252255dd-e572-43c1-a06c-41de6de2e8dc.png)

However, when the code type is not JavaScript, the page will display an error

![Snipaste_2022-08-12_21-43-29](https://user-images.githubusercontent.com/55755493/184366218-c4eeeb08-50db-4def-9a98-5a96a0c22c4c.png)

When there is an error in the display, view it through the browser developer tool. The HTML structure is

![Snipaste_2022-08-12_21-45-21](https://user-images.githubusercontent.com/55755493/184366521-f33875f6-f36a-4561-819d-ff8818569faa.png)

But right click "view web page source code" on the page，The display of the web page source code is different from that of the page display (console HTML viewer)
![Snipaste_2022-08-12_21-46-51](https://user-images.githubusercontent.com/55755493/184366776-9272f346-30a4-4acc-902f-b94e145029f9.png)
![Snipaste_2022-08-12_21-48-27](https://user-images.githubusercontent.com/55755493/184367024-56a5b0c7-f84f-4efc-93a1-7875db0801cf.png)

**Comment by danielroe** [comment]
This doesn't seem like a Nuxt issue, but rather how you are using marked (or an issue within marked). Please feel free to provide a reproduction, which may help people advise you.
